public/index.html

```diff
@@ -1,54 +1,322 @@
+          <a class="btn_header login_btn" href="#">
+            <svg class="icon icon_sign_in">
+              <use xlink:href="#sign_in">
+                <symbol id="sign_in" xmls="#" viewBox="#">
+                  <path d="#"></path>
+                  <path d="#"></path>
+                </symbol>
+              </use>
+            </svg>
+            وارد شوید
+          </a>
+        </div>
+
+        <div class="top_menu">
+          <div class="container">
+            <div class="rowflex">
+              <div class="menu_right_header">
+                <ul class="menuTopHeaderDigi">
+                  <li>
+                    <a title="خانه" href="#"></a>
+                  </li>
+                  <li>
+                    <a title="دانلود فیلم" href="#">
+                      <div class="icon_holder">
+                        <svg class="icon icon_film_reel"></svg>
+                      </div>
+                      <div class="text_holder">
+                        <span>دانلود فیلم</span>
+                        <small>MOVIES</small>
+                      </div>
+                    </a>
+                    <ul class="TopmenuSubmenu">
+                      <li class="innerSubmenu">
+                        <div class="rightMenuItems">
+                          <nav class="#">
+                            <ul id="#" class="menu">
+                              <li id="#" class="#">
+                                <a href="#">دانلود فیلم</a>
+                              </li>
+                              <li id="#" class="#">
+                                <a href="#">فیلم های دارای پخش آنلاین</a>
+                              </li>
+                              <li id="#" class="#">
+                                <a href="#">250 فیلم برتر تاریخ</a>
+                              </li>
+                              <li id="#" class="#">
+                                <a href="#">دانلود انیمیشن</a>
+                              </li>
+                              <li id="#" class="#">
+                                <a href="#">جدول باکس آفیس</a>
+                              </li>
+                              <li id="#" class="#">
+                                <a href="#">فیلم های دوبله فارسی</a>
+                              </li>
+                              <li id="#" class="#">
+                                <a href="#">فیلم های بزودی</a>
+                              </li>
+                              <li id="#" class="#">
+                                <a href="#">سینمای هند</a>
+                              </li>
+                              <li id="#" class="#">
+                                <a href="#">فیلم های 3 بعدی</a>
+                              </li>
+                            </ul>
+                          </nav>
+                        </div>
+                        <div class="MenuImageHolder">
+                          <img src="#" al="MoviesSubMenu">
```

Comment:
```suggestion
                          <img src="" al="MoviesSubMenu">
```

let's keep their `src` attribute empty until we get some images